customHttp.yml

```diff
@@ -0,0 +1,19 @@
+customHeaders:
+  - pattern: '**'
+    headers:
+      - key: 'Access-Control-Allow-Origin'
+        value: '*'
+  - pattern: '/v2/kommunicate.app'
+    headers:
+      - key: 'Content-Type'
+        value: 'text/javascript'
+      - key: 'Cache-Control'
+        value: 'public, max-age=300'
+  - pattern: '*.html'
+    headers:
+      - key: 'Cache-Control'
+        value: 'public, max-age=300'
```

Comment:
if it got this right, this means we will cache these files for 5 minutes.
so our deployments will take 5 minutes to reflect?

Reply:
Yes. It will take 5 mins.

Reply:
Alright. Thanks.
cc: @devashishmamgain i hope that is fine.

Reply:
yes this is fine, we use usually do not change this file but keeping it as 5 mins in case if we ever fix any issue in this.